API.Tests/PostProjectsModelValidatorTests.cs

```diff
@@ -77,7 +78,7 @@ public void ProjectStatus_Is_Required()
 
             model = new PostProjectsRequestModel
             {
-                ProjectStatus = 8
+                ProjectStatus = (Models.Upstream.Enums.ProjectStatusEnum)8
```

Comment:
Question: Is the cast necessary? Might be cleaner to just use Models.Upstream.Enums.ProjectStatusEnum.Complete for example.

Warning: The upstream enum only seems to contain 2 values and 8 isn't one of them.  This cast might cause the test to fail.

    public enum ProjectStatusEnum
    {
        InProgress = 1,
        Completed = 2
    }

Reply:
Just expanded the file and noticed that this test is looking for an error so ignore my warning.

Reply:
This test ensures that invalid Enum values are validated and that an error is raised when that happens.